src/lib/components/Navbar/NavbarBrand.tsx

```diff
@@ -8,16 +9,24 @@ export interface FlowbiteNavbarBrandTheme {
   base: string;
 }
 
-export interface NavbarBrandProps extends PropsWithChildren<ComponentProps<'a'>> {
+export interface NavbarBrandProps extends PropsWithChildren<ComponentProps<'a'>>, Partial<Pick<LinkProps, 'to'>> {
```

Comment:
is `to` optional here?

Reply:
yes, you can use both `a` tag and `Link` component from react-router-dom.